src/lib/components/Search/index.js

```diff
@@ -165,128 +170,26 @@ class Search extends BaseStateElement {
-        </div>
-      `;
-    }
-
-    return html`
-      <div class="web-search-popout">
-        <div class="web-search-popout__heading">Pages</div>
-        <ul
-          id="web-search-popout__list"
-          class="web-search-popout__list"
-          role="listbox"
-        >
-          ${this.itemsTemplate}
-        </ul>
-      </div>
-    `;
-  }
-
-  get itemsTemplate() {
-    // Note that our anchors have tabindex=-1 to prevent them from
-    // being focused.
-    // This is intentional because focus needs to stay in the input field.
-    // When the user is pressing arrow keys, we use a virtual cursor and
-    // aria-activedescendant to indicate the active anchor.
-    return this.hits.map((hit, idx) => {
-      if (!hit._highlightResult.title || !hit._highlightResult.title.value) {
-        return html``;
-      }
-
-      let title = hit._highlightResult.title.value;
-      // Escape any html entities in the title except for <strong> tags.
-      // Algolia sends back <strong> tags in the title which help highlight
-      // the characters that match what the user has typed.
-      title = allowHtml(escapeHtml(title), 'strong');
-      // Strip backticks as they look a bit ugly in the results.
-      title = title.replace(/`/g, '');
-      return html`
-        <li class="web-search-popout__item">
-          <a
-            id="web-search-popout__link--${idx}"
-            class="web-search-popout__link ${idx === this.cursor
-              ? 'web-search-popout__link--active'
-              : ''}"
-            aria-selected="${idx === this.cursor}"
-            tabindex="-1"
-            href="${hit.url}"
-            >${unsafeHTML(title)}</a
-          >
-        </li>
-      `;
-    });
-  }
-  /* eslint-enable indent */
-
   firstUpdated() {
     /** @type HTMLInputElement */
     this.inputEl = this.renderRoot.querySelector('.web-search__input');
   }
 
   /**
-   * Keep track of cursor changes and reflect them to aria-activedescendant.
-   * This ensures screen readers properly announce the current search result.
-   * We do this because focus never leaves the search input box, so when the
-   * user is arrowing through results, we have to tell the screen reader about
-   * it.
+   * Passes on updated search properties to the search results element.
    * @param {Map} changedProperties A Map of LitElement properties that changed.
    */
   updated(changedProperties) {
-    if (!changedProperties.has('cursor')) {
-      return;
-    }
-
-    if (this.cursor === -1) {
-      this.inputEl.removeAttribute('aria-activedescendant');
-      return;
-    }
-
-    this.inputEl.setAttribute(
-      'aria-activedescendant',
-      `web-search-popout__link--${this.cursor}`,
-    );
+    // Reflect changed properties to the results el.
+    const sharedProperties = ['query', 'hits', 'showHits'];
+    sharedProperties.forEach((property) => {
+      if (changedProperties.has(property)) {
+        this.resultsEl[property] = this[property];
```

Comment:
I think we have to do a `if (this.resultsEl)` check here as well... pretty much anytime we want to access it we probably need to guard against a case where it doesn't exist.